Tools/Scripts/run-jsc-stress-tests

```diff
@@ -589,6 +589,18 @@ $isFTLPlatform = !($architecture == "x86" || $architecture == "arm" || $architec
 $isWasmPlatform = $isFTLPlatform || $architecture == "arm"
 $isSIMDPlatform = $architecture == "arm64" || $architecture == "x86_64"
 
+# This is meant for skipping execution of tests than require a lot of address
+# space. Cf. $memoryLimited, which is meant for tests that actually make use of
+# a lot of memory.
+# Technically, this should be the bits available in userspace, but right now we
+# don't need to be this precise.
+$addressBits = case $architecture
+               when /.*64.*/
+                   48
```

Comment:
nit: iOS is 36-bit. I'm not sure it matters here

Reply:
Did not know that. Changed the line to 36 instead -- we can worry about differentiating between 64-bit iOS and non-iOS when we get tests that require > 36 bits of address space.